addons/account/static/src/js/reconciliation/reconciliation_renderer.js

```diff
@@ -389,9 +389,9 @@ var LineRenderer = Widget.extend(FieldManagerMixin, {
                     .attr("data-content", qweb.render('reconciliation.line.mv_line.details', {'line': line}));
             }
             if (line.already_paid === false &&
-                ((state.balance.amount_currency < 0 || line.partial_reconcile)
+                (((state.balance.amount_currency < 0 || line.partial_reconcile)
                     && line.amount > 0 && state.st_line.amount > 0 && targetLineAmount < line.amount && partialDebitProps <= 1) ||
-                ((state.balance.amount_currency > 0 || line.partial_reconcile)
+                ((state.balance.amount_currency > 0 || line.partial_reconcile))
```

Comment:
Are sure about the position of the extra closing parenthesis? shouldn't it be on next line?

Reply:
jesus -_-

Reply:
https://github.com/odoo/odoo/pull/29874